src/ProtocolHandler/HTTP/Router/ChorizoRouter.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+using Chorizo.ProtocolHandler.HTTP.Requests;
+using Chorizo.ProtocolHandler.HTTP.Responses;
+
+namespace Chorizo.ProtocolHandler.HTTP.Router
+{
+    public class ChorizoRouter:IRouter
+    {
+        private readonly Dictionary<string, Route> _routes;
+        public ChorizoRouter()
+        {
+            _routes = new Dictionary<string, Route>();
+        }
+
+        public void Match(IRequest req, IResponse res)
+        {
+            if (_routes.ContainsKey(req.Path()) && _routes[req.Path()].HttpMethod() == req.Method())
+            {
+                _routes[req.Path()].Go(req, res);
+            }
+            else
+            {
+                res.Status(404).Send("Not Found");
```

Comment:
Is there a reason why you didn't use the functionality from `ResponseCodes` here?

Reply:
When `Status` is called, it takes the integer you pass it and uses `ResponseCodes` to set the associated `statusText` in the Response. Using `ResponseCodes` to set the text of the response body seems like it may cause confusion of what `Send` does.